docs/UsersGuide/source/ContainerQuickstart.rst

```diff
@@ -119,17 +117,10 @@ On non-Level 1 systems, users should build the container in a writable sandbox:
 
 .. code-block:: console
 
-   sudo singularity build --sandbox ubuntu20.04-intel-srwapp docker://noaaepic/ubuntu20.04-intel-srwapp:develop
-
-Some users may prefer to issue the command without the ``sudo`` prefix. Whether ``sudo`` is required is system-dependent. 
+   sudo singularity build --sandbox ubuntu20.04-intel-srwapp docker://noaaepic/ubuntu20.04-intel-srwapp:release-public-v2.1.0
 
 .. note::
-   Users can choose to build a release version of the container (SRW App v2.0.0) using a similar command:
-
-   .. code-block:: console
-
-      sudo singularity build --sandbox ubuntu20.04-intel-srwapp docker://noaaepic/ubuntu20.04-intel22-ufs-srwapp:release-public-v2
-
+   Some users may prefer to issue the command without the ``sudo`` prefix. Whether ``sudo`` is required is system-dependent. 
 
```

Comment:
@JeffBeck-NOAA Do you mean the part where it says "Users can choose to build a release version of the container (SRW App v2.0.0) using a similar command..."? If so, I can add that. It's available in the develop branch docs tho. I figured users working on develop might be interested in knowing there's a static release version, but I kind of assume that users on the release branch are there for a reason (i.e., new to UFS or looking for static code), since it's not the default branch.

Reply:
Sorry!  I got confused.  I think it's fine as is.  I suppose the v2.0.0 container language can be updated with v2.1.0 in the develop branch rst file?

Reply:
Yes!